content/en/docs/reference/using-api/deprecation-policy.md

```diff
@@ -365,6 +365,42 @@ This applies only to significant, user-visible behaviors which impact the
 correctness of applications running on Kubernetes or that impact the
 administration of Kubernetes clusters, and which are being removed entirely.
 
+The exception to the above rule are **feature-gates**. Feature gates are a set of
+key=value pairs that enable users to enable/disable experimental features.
+Features can be in Alpha, Beta or GA stage.
+Due to the particular nature of feature gates we recommend that when features
+hit GA, the correspondent feature gate be dropped.
```

Comment:
This is unclear - naive reading say that as soon as a feature hits GA the gate is removed.  I would instead say something like:

```
Feature gates are intended to cover the development of a feature - they are not intended to be long-term APIs.  As such, they are expected to be deprecated and removed after a feature becomes GA.  Some features will want long-term enable/disable flags - those flags are part of the feature and should themselves be covered by the feature gate.

In general, when a feature becomes GA and default, the associated feature flag will be announced as deprecated and the effect of that feature gate will become a no-op.  This allows configurations which set the gate to function through the transition.  In some cases, the old behavior requires a considerable amount of time before it can be removed. In these cases the feature gate's behavior can be retained until the old behavior is removed, at which point the feature gate can be deprecated as above.  

Feature gates are not versioned in the same way as the previously discussed components,￼ therefore the rules for deprecation are as follows:
```

What do you think?